From 26eca73b4fb433569ce108e1f6b0a2160b26369f Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Tue, 26 Nov 2013 16:52:49 +0800
Subject: [PATCH 1/4] ENGR00215174 update to pass build for android 4.4

evk_6sl not support 3D, so remote it from build for evk_6sl

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Android.mk b/Android.mk
index 0aeee74..7e97fbf 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,4 @@
+ifeq ($(HAVE_FSL_IMX_GPU3D),true)
 LOCAL_PATH:= $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -39,3 +40,4 @@ endif
 include $(BUILD_PACKAGE)
 
 include $(call all-makefiles-under, $(LOCAL_PATH))
+endif
-- 
1.8.0

